src/Stripe.net/Services/InvoiceItems/StripeInvoiceItemUpdateOptions.cs

```diff
@@ -11,13 +11,16 @@ public class StripeInvoiceItemUpdateOptions : StripeBaseOptions, ISupportMetadat
         [JsonProperty("description")]
         public string Description { get; set; }
 
-        [JsonProperty("currency")]
```

Comment:
This is not valid per https://stripe.com/docs/api#update_invoiceitem which means it never worked so not sure if a breaking change

Reply:
If it never worked, then it's not a breaking change IMO.